Dockerfile

```
@@ -2,12 +2,13 @@ FROM alpine
 
 LABEL maintainer Bill Wang <ozbillwang@gmail.com>
 
-RUN apk --update add git openssh && \
-    rm -rf /var/lib/apt/lists/* && \
-    rm /var/cache/apk/*
+COPY docker-entrypoint.sh /docker-entrypoint.sh
+
+RUN apk --update --no-cache add git openssh-client && \
```

Keeping arguments in alphabetical order, makes things tidy
```suggestion
RUN apk --no-cache --update add git openssh-client
```